src/javafundamentals/streams/Ex06MaybeInThere.java

```diff
@@ -8,4 +12,10 @@
  */
 
 public class Ex06MaybeInThere {
+    public static void main(String[] args) {
+        IntStream randomStream = new Random().ints(10, 1, 10);
+        OptionalInt any = randomStream.findAny();
+        any.ifPresent(i-> System.out.println(i));
```

Comment:
This line is ok, You are saying to the `any` optional , that if it has any value, then use that lambda to print the value.